contracts/libraries/Pool.sol

```diff
@@ -74,8 +74,9 @@ library Pool {
         int24 tick;
         uint24 protocolFees;
         uint24 hookFees;
+        // used for the swap fee, either static at initialize or updateable to the dynamic value
```

Comment:
"updateable to the dynamic value" a bit confusing - maybe just "or dynamic via hook"